datatransfer/impl/graphsync/graphsync.go

```diff
@@ -197,7 +205,7 @@ func (impl *graphsyncImpl) notifySubscribers(evt datatransfer.Event, cs datatran
 
 // get all in progress transfers
 func (impl *graphsyncImpl) InProgressChannels() map[datatransfer.ChannelID]datatransfer.ChannelState {
-	return nil
+	return impl.channels
```

Comment:
this is actually the correct response for this function so yay on that... (though it will need to be persisted eventually)